Content.Shared/Body/Systems/SharedBodySystem.cs

```diff
@@ -50,7 +50,7 @@ public override void Initialize()
         // This is blursed
         var slotIndex = containerSlotId.IndexOf(PartSlotContainerIdPrefix, StringComparison.Ordinal);
 
-        if (slotIndex < -1)
+        if (slotIndex < 0)
```

Comment:
if slot 0 is the root part's index, then add a comment here clarifying that.

Reply:
I'm not sure what you mean by that.
Isn't this just a check to see if the string search above actually found anything?
(``slotIndex`` is just the index of the substring "body_part_slot_" in ``containerSlotId``)

Not sure what that has to do with the root part index since the function seems to just strip "body_part_slot_" out of the string it takes in.